scribus/text/textshaper.cpp

```diff
@@ -14,6 +13,7 @@
 #include "storytext.h"
 #include "styles/paragraphstyle.h"
 
+BreakIterator* TextShaper::m_lineIterator = NULL;
```

Comment:
Move this to story text like the other iterators, make `getLineIterator()` static and call it from there.